src/components/ToDoList/Todo.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { TodoPropsType } from './types';
+import EditTodoForm from './EditTodoForm';
+
+function Todo(props: TodoPropsType): JSX.Element {
+    const { task, deleteTodo, editTodo, editTask, toggleComplete } = props;
+
+    return (
+        <div className="flex relative justify-between">
+            <span className="flex items-center py-3 px-1 mb-4 cursor-grab text-purple-400">
+                <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    className="w-3 h-3"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                >
+                    <path
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        strokeWidth="2"
+                        d="M4 6h16M4 12h16M4 18h16"
+                    />
+                </svg>
+            </span>
+            {task.isEditing ? (
+                <EditTodoForm key={task.id} editTask={editTask} task={task} />
+            ) : (
+                <div
+                    className="flex justify-between w-full text-center bg-purple-500 text-white py-3 px-4 rounded-md mb-4 cursor-pointer"
+                    onDoubleClick={() =>
+                        !task.completed ? editTodo(task.id) : {}
```

Comment:
move to function changeCompletedStatus or something like that

Reply:
deleted, because it's not in use